### ISSUE TITLE
Mitglieder müssen in Mitgliederliste einzeln ausgewählt werden

### ISSUE THREAD

**Issue by narrenfrei** [heimrichhannot/contao-member_plus#7]
Bin mir nicht sicher ob Feature oder Bug, aber ich muss beim Elementtyp Mitgliederliste alle Mitglieder einzeln auswählen. Das aktivieren der Mitgliedergruppen alleine reicht nicht aus.

Außerdem hat die Bereich Mitglieder einen falschen Hilfetext.

**Comment by dhe** [comment]
Yes, it would great to add the possibility to select the members itself or just select a group, than all members would selected. A order field would also be nice

thx
